python/glow/gwas/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .linear_regression import *
+__all__ = ['linear_regression']
```

Comment:
This is a way to accomplish what I mentioned about making the docs show up as `glow.gwas.linear_regression` instead of `glow.gwas.linear_regression.linear_regression`.